src/ripple/app/paths/RippleCalc.cpp

```diff
@@ -149,7 +149,12 @@ RippleCalc::Output RippleCalc::rippleCalculate (
         {
             JLOG (j.error()) << "Exception from flow: " << e.what ();
             if (!useFlowV1Output)
-                Rethrow();
+            {
+                // return a tec so the tx is stored
+                path::RippleCalc::Output exceptResult;
+                exceptResult.setResult(tecINTERNAL);
+                return exceptResult;
+            }
```

Comment:
The exception is thrown from flowV2. If we're actually using flowV1, the fact that flowV2 threw should effect the result. So it's kosher.

Note this is only used in a special compare mode, and in normal operation only flow v1 (when replaying old ledgers) or flow v2 (for new ledgers) will run.